### PR TITLE
added consant for inventory limit

### LINKED ISSUE #291: Use named constant for `inventoryLimit` of `-1`
As discussed in [this comment thread](https://github.com/jeremyckahn/farmhand/pull/289/files#r894128749), an `inventoryLimit` value of `-1` is equivalent to infinite inventory space. It's hardcoded as a magic number [throughout the codebase](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/jeremyckahn/farmhand%24+inventoryLimit.*-1&patternType=regexp), but it should be refactored to be a named constant in `constants.js`.

### PR DESCRIPTION
**Pull request by kvaithin**
### What this PR does
Close #291 

Updated to use constant for inventory limit value.

### How this change can be validated
No functionality change. Refactor to reuse constant inventory limit value.

### CODE REVIEW

**Review by jeremyckahn — CHANGES_REQUESTED** [review]
Hi @kvaithin, thanks for picking up #291 and opening this PR! I really appreciate the help. 

The code here generally looks good, but I think this is going to need to wait until #289 is merged. These PRs are touching a lot of the same code, and it'll be less risky to get that in and then either rebase your branch on top of `develop` or `git merge` it in.

Finishing #289 is my top priority, so hopefully you won't have to wait too long. Sorry for the inconvenience. I didn't think anyone would pick up #291 so quickly! 😅

**Comment by kvaithin** [comment]
> Hi @kvaithin, thanks for picking up #291 and opening this PR! I really appreciate the help.
> 
> The code here generally looks good, but I think this is going to need to wait until #289 is merged. These PRs are touching a lot of the same code, and it'll be less risky to get that in and then either rebase your branch on top of `develop` or `git merge` it in.
> 
> Finishing #289 is my top priority, so hopefully you won't have to wait too long. Sorry for the inconvenience. I didn't think anyone would pick up #291 so quickly! 😅

Sure, will keep a lookout for #289 completion. 👍

File: src/constants.js
Comment (kvaithin) [reply src/constants.js]: Sounds good. Updated to `INFINITE_STORAGE_LIMIT`.

**Comment by jeremyckahn** [comment]
https://github.com/jeremyckahn/farmhand/pull/289 has just been merged! Sorry for the conflicts you'll have to deal with @kvaithin. In case it's easier just to start fresh, feel free to close this PR and open a new one. Let me know if I can help you at all!

**Comment by kvaithin** [comment]
> #289 has just been merged! Sorry for the conflicts you'll have to deal with @kvaithin. In case it's easier just to start fresh, feel free to close this PR and open a new one. Let me know if I can help you at all!

Np, closing this PR. Raised #293

**Pull request closed without merging**
